src/components/Feedback/Toast/toast.tsx

```diff
@@ -0,0 +1,169 @@
+            ];
+            return newGrid;
+        });
+    
+        if (timeout > 0) {
+            const timerID = setInterval(() => {
+                setTimers((prev) => {
+                    const next = { ...prev }; // copy
+                    if (next[id] >= INTERVALS) {
+                        removeItem(position, id);
+                        clearInterval(timerID);
+                        delete next[id];
+                    } else {
+                        next[id] = (next[id] ?? 0) + 1;
+                    }
+                    return next;
+                });
+            }, timeout / INTERVALS);
+            setTimers((prev) => ({ ...prev, [id]: 0 }));
+        }
+    }
+
+    // Remove item by index
+    function removeItem(position: Position, id: string) {
+        const [_, setter] = grid[position];
+
+        setter((prev) => {
+            const newGrid = [...prev];
+            return newGrid.filter((item) => item.id !== id);
+        });
+    }
+
+    const Toast: ParentComponent = () => {
+        const flexItems = (number: number): 'start' | 'center' | 'end' => {
+            switch (number) {
+                case 0:
+                    return 'start';
+                case 1:
+                    return 'center';
+                case 2:
+                    return 'end';
+                default:
+                    return 'start';
+            }
+        };
+
+        const direction = (row: number) => {
+            return row === 2 ? 'column-reverse' : 'column';
+        };
+
+        return (
+            <Portal mount={document.querySelector('#root')!}>
+                <Grid
+                    style={{
+                        width: '100vw',
+                        height: '100vh',
+                        position: 'absolute',
+                        'z-index': 9999,
+                        'pointer-events': 'none',
+                    }}
+                    cols={3}
+                    rows={3}
+                >
+                    <For each={positions}>
+                        {(pos, index) => {
+                            const [getter] = grid[pos];
+                            return (
+                                <GridTile row={Math.floor(index() / 3) + 1} col={(index() % 3) + 1}>
+                                    <Flex
+                                        justify-content={flexItems(Math.floor(index() / 3))}
+                                        align-items={flexItems(index() % 3)}
+                                        direction={direction(Math.floor(index() / 3))}
+                                        style={{ height: '100%' }}
+                                    >
+                                        <For each={getter()}>
+                                            {(item) => {
+                                                return (
+                                                    <div class="toast-item" style={{ 'pointer-events': 'auto' }}>

```

Comment:
```suggestion
                                                    <div style={{ 'pointer-events': 'auto' }}>
```
Do we need the `toast-item` class? I don't see it being used anywhere